### ISSUE TITLE
Using CreateSignedUrl with TransformOptions fails

### ISSUE THREAD

**Issue by alustrement-bob** [supabase-community/storage-csharp#15]
# Bug report

## Describe the bug
When creating a signed URL with TransformOptions :
```
var signedUrl = await supabase.Storage
  .From("bucket")
  .CreateSignedUrl("file.jpg", 3000, new Supabase.Storage.TransformOptions
    {
      Width = 800,
      Height = 300,
      Resize = Supabase.Storage.TransformOptions.ResizeType.Contain
    }
  );
```
This leads to this error :
```
Unhandled exception. Supabase.Storage.Exceptions.SupabaseStorageException: body/transform/resize must be equal to one of the allowed values
   at Supabase.Storage.Helpers.MakeRequest(HttpMethod method, String url, Object data, Dictionary`2 headers, CancellationToken cancellationToken)
   at Supabase.Storage.Helpers.MakeRequest[T](HttpMethod method, String url, Object data, Dictionary`2 headers)
   at Supabase.Storage.StorageFileApi.CreateSignedUrl(String path, Int32 expiresIn, TransformOptions transformOptions)
```

## Additional context

- The exact same error is also raised when the 'Resize' parameter is not provided
- No error when TransformOptions is null
- Give TransformOptions works perfectly with the Download function :
```
var bytes = await supabase.Storage
  .From("bucket")
  .Download("file.jpg", new TransformOptions
    {
      Width = 800,
      Height = 300,
      Resize = TransformOptions.ResizeType.Contain
    }
  );
```
- The supabase-js equivalent works great :
```
await supabase.storage
  .from('bucket')
  .createSignedUrl('file.jpg', 3000, {
    transform: {
      width: 800,
      height: 300,
      resize: 'contain'
    },
  })
  .then(async ({ data, error }) => {
    if (error) {
      console.error(`Error: ${error.message} ${error.stack}`);
    } else {
      const response = await fetch(data.signedUrl);
      const buffer = await response.arrayBuffer();
      fs.writeFileSync('out.jpg', Buffer.from(buffer));
      console.log('ok');
    }
  });
```

**Comment by acupofjose** [comment]
Available in v2.0.1 - thanks!

**Comment by alustrement-bob** [comment]
Thanks ! Any chance to have an update of the supabase-csharp nuget package right away ?

**Comment by acupofjose** [comment]
I’ve got a couple other commits to bundle in, but should be in the next couple hours or so

**Comment by acupofjose** [comment]
Should be available in v1.0.2!
